addons/mail/static/src/models/rtc_call_viewer/rtc_call_viewer.js

```diff
@@ -88,30 +88,28 @@ function factory(dependencies) {
             this.messaging.userSetting.rtcConfigurationMenu.toggle();
         }
 
-        /**
-         * @param {boolean} force Force the fullScreen state.
-         */
-        async toggleFullScreen(force) {
-            if (!this.exists()) {
-                return;
-            }
+        async activateFullScreen() {
             const el = document.body;
-            const fullScreenElement = document.webkitFullscreenElement || document.fullscreenElement;
-            if (force !== undefined ? force : !fullScreenElement) {
-                try {
-                    if (el.requestFullscreen) {
-                        await el.requestFullscreen();
-                    } else if (el.mozRequestFullScreen) {
-                        await el.mozRequestFullScreen();
-                    } else if (el.webkitRequestFullscreen) {
-                        await el.webkitRequestFullscreen();
-                    }
+            try {
+                if (el.requestFullscreen) {
+                    await el.requestFullscreen();
+                } else if (el.mozRequestFullScreen) {
+                    await el.mozRequestFullScreen();
+                } else if (el.webkitRequestFullscreen) {
+                    await el.webkitRequestFullscreen();
+                }
+                if (this.exists()) {
                     this.update({ isFullScreen: true });
-                } catch (e) {
+                }
+            } catch (e) {
+                if (this.exists()) {
```

Comment:
Ideally we should tell the user that full screen was not possible (and why if possible), maybe with a popup notification?